aptos-node/src/lib.rs

```diff
@@ -61,6 +61,10 @@ pub struct AptosNodeArgs {
     #[clap(long, parse(from_os_str), requires("test"))]
     test_dir: Option<PathBuf>,
 
+    /// Path to node configuration file override for local test mode. Cannot be used with --config
+    #[clap(long, parse(from_os_str), requires("test"), conflicts_with("config"))]
+    config_override: Option<PathBuf>,
```

Comment:
Can we name this `--test-config-override`? AFAICT, this is only relevant for the test mode, right? Because the normal mode will do this by default (i.e., you specify the config options in the file and anything you don't specify is chosen by the default or config optimizer). Does that make sense?

Reply:
yea makes sense!